Algorithm/QuantConnect.Algorithm.csproj

```diff
@@ -115,6 +124,13 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
+  <Import Project="..\packages\AlexCatarino.PythonNET.Runtime.1.0.9\build\AlexCatarino.PythonNET.Runtime.targets" Condition="Exists('..\packages\AlexCatarino.PythonNET.Runtime.1.0.9\build\AlexCatarino.PythonNET.Runtime.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('..\packages\AlexCatarino.PythonNET.Runtime.1.0.9\build\AlexCatarino.PythonNET.Runtime.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\AlexCatarino.PythonNET.Runtime.1.0.9\build\AlexCatarino.PythonNET.Runtime.targets'))" />
```

Comment:
Same with this change block; make sure to reference the official QC-Pythonnet fork not your personal repo.